index.html

```diff
@@ -13,7 +13,7 @@
 				<a href="/tea">Tea</a>
 			</nav>
 			<main>
-				Текст по инфастилю
+				Текст по инфостилю
```

Comment:
Please typograf it

Reply:
In what sense?

Reply:
> In what sense?

[Typograf](https://www.artlebedev.ru/typograf/)